src/App.js

```diff
@@ -5,11 +5,14 @@ import Signup from './Components/Signup/Signup';
 import Topbar from './Layouts/Topbar';
 import IndexRouter from './Routes/Route';
 
+
 function App() {
   return (
 
-     <IndexRouter />
+<IndexRouter />
```

Comment:
remove the extra spaces make it return <IndexRouter/>

Reply:
spaces  are removed